README.md

```diff
@@ -87,7 +87,7 @@ npm run build
 
 To start the application you can run
 ```
-node index.js
+node dist/index.js
```

Comment:
Propose using `npm start` so to avoid declaring the app start script twice.
```suggestion
npm start
```

Reply:
@rahulk94 good suggestion. Looks like `npm start` already present in the readme [here](https://github.com/jwu910/check-it-out/blob/typescript-migration/README.md?plain=1#L94), so this would be a straight removal. Would you be willing to file an issue for this? It would apply just as well to the main branch as to this one.

Reply:
Yep sure I'll open an issue + PR

Reply:
Oh good point. We can link npm start to the build assets. 👌

Reply:
https://github.com/jwu910/check-it-out/pull/533

Reply:
Thanks for raising this, took a look at concluded it's not really an issue. Readme offers flexibility and not forces one method over the other. We'll leave this in for now 👌🏻